Aquarius/Src/Aquarius/Sound/Devices.cpp

```diff
@@ -0,0 +1,76 @@
+#include "Devices.h"
+#include "Aquarius/Core/Log.h"
+#include <AL/alc.h>
+
+
+namespace Aquarius {
+
+    namespace Sound {
+
+        Device::Device()
+        {
+            Initialize();
+        }
+
+        void Device::Initialize()
+        {
+            m_Device = alcOpenDevice(nullptr);
+
+            if (!m_Device)
+            {
+                AQ_CORE_ERROR("Failed to open a sound device");
```

Comment:
If the device doesn't open, you probably wanna return from the function here, if not I feel like something undesirable might happen below when you call `alcCreateContext`